1.10/requirements.txt

```diff
@@ -1,4 +1,4 @@
-gunicorn==19.7.1
+gunicorn==19.8.1
 django==1.10.8
 psycopg2==2.7.4 --no-binary psycopg2
-gevent==1.2.2
+gevent==1.3.2.post0
```

Comment:
Just to confirm, you're using the `1.3.2.post0` package because the `1.3.2` one was busted?

Reply:
Yeah. There was a [packaging error](http://www.gevent.org/changelog.html#post0-2018-05-30) I didn't dive too deep into.